pyproject.toml

```diff
@@ -225,7 +225,7 @@ python_version = 3.11
 warn_unused_configs = true
 warn_redundant_casts = true
 warn_unused_ignores = true
-
+check_untyped_defs = true
```

Comment:
The main goal of this:

[Type-checks the interior of functions without type annotations.
](https://mypy.readthedocs.io/en/stable/config_file.html)